docs/svelte-testing-library/intro.md

```diff
@@ -48,7 +48,7 @@ App.spec.js
 
 ```javascript
 import App from '../src/App.svelte'
-import { render, cleanup } from 'svlt-testing-library'
+import { render, cleanup } from 'svelte-testing-library'
```

Comment:
Is there coordination between package authors for this change?

Reply:
@kentcdodds yep. Wendell had published an empty package with the name and didn't have time to implement it so he added me as a maintainer.  It's all on Twitter. 🙂. https://twitter.com/benmonro/status/1124339261853954048?s=21

Reply:
Most excellent!